setup.py

```diff
@@ -46,18 +46,19 @@
     ],
     extras_require={
         # use `pip install EthicML[dev]` to install development packages
-        'dev': [
+        "dev": [
             # "black",
             # "mypy >= 0.720",
             # "pylint >= 2.0",
             "pytest >= 3.3.2",
             "pytest-cov >= 2.6.0",
             "torch >= 1.1.0, <= 1.1.0.post2",
+            "pre-commit",
         ]
```

Comment:
I'm starting to think we should have it actually like this:

```python
        'ci': [
            "pytest >= 3.3.2",
            "pytest-cov >= 2.6.0",
            "torch >= 1.1.0, <= 1.1.0.post2",
        ],
        'dev': [
            "black",
            "mypy >= 0.720",
            "pylint >= 2.0",
            "pytest >= 3.3.2",
            "pytest-cov >= 2.6.0",
            "pre-commit",
        ]
```

and then on the CI we install ethicml with `pip install .[ci]` and locally you can install it with `pip install ethicml[dev]` if you want.

Reply:
torch just in the ci?

Reply:
Yeah, the way I'm thinking about it is that `dev` is for the human developer who might be on Windows where `pip install torch` doesn't work.